From bugzilla-daemon at main.slony.info  Mon Apr  1 06:12:08 2013
From: bugzilla-daemon at main.slony.info (bugzilla-daemon at main.slony.info)
Date: Mon,  1 Apr 2013 06:12:08 -0700 (PDT)
Subject: [Slony1-bugs] [Bug 288] Error occurred when creating new slony-i
	cluster
In-Reply-To: <bug-288-4@http.www.slony.info/bugzilla/>
References: <bug-288-4@http.www.slony.info/bugzilla/>
Message-ID: <20130401131208.7A0A4290D2D@main.slony.info>

http://www.slony.info/bugzilla/show_bug.cgi?id=288

--- Comment #1 from Steve Singer <ssinger at ca.afilias.info> 2013-04-01 06:12:08 PDT ---
You don't tell us exactly which version of slony you are using and exactly
which version of pgadmin you are using.

It is possible that the version of slony you have installed isn't compatible
with the version of pgadmin you are using.  The slony team doesn't
maintain/support using slony through pgadmin (the pgadmin team does that). 

We encourage people to configure their slony clusters by using slonik directly
or tools that call slonik (ie the altperl scripts)

-- 
Configure bugmail: http://www.slony.info/bugzilla/userprefs.cgi?tab=email
------- You are receiving this mail because: -------
You are on the CC list for the bug.
You are the assignee for the bug.

